import { TryCatch } from "../Helpers/Error.js";

export const ReviewController = {
    createNewReview:TryCatch(async(req,res)=>{

    }),
    getAllReviews:TryCatch(async(req,res)=>{

    }),
    getReviewById:TryCatch(async(req,res)=>{

    }),
    editReview:TryCatch(async(req,res)=>{

    }),
    deleteReview:TryCatch(async(req,res)=>{

    }),
}